src/diffpy/utils/tools.py

```diff
@@ -65,7 +64,7 @@ def load_config(file_path):
             config = json.load(f)
         return config
     else:
-        return None
+        return {}
 
 
 def _sorted_merge(*dicts):
```

Comment:
I wonder do we want to make some of the functions private (like `load_config`)?

Reply:
yes, this should probably be private, I agree.

Can we also check?  I don't think we need `_sorted_dict()` any more so that could be deleted on a new PR.

Reply:
Yes, I can fix these on a new PR, once that updater workflow PR is merged